mempool/src/pool/store.rs

```diff
@@ -77,10 +85,21 @@ pub struct MempoolStore {
     pub(super) spender_txs: BTreeMap<OutPoint, Id<Transaction>>,
 }
 
+#[derive(Clone, Copy, PartialEq, Eq)]
+pub(super) enum MempoolRemovalReason {
+    Block,
+    //Conflict,
+    Expiry,
+    SizeLimit,
+    //Reorg,
```

Comment:
Should the commented lines be removed?

Reply:
We'll eventually use these as well, but I guess we can delete them for now.